client/core/core.go

```diff
@@ -1473,6 +1482,8 @@ func (c *Core) Cancel(pw []byte, tradeID string) error {
 	var result = new(msgjson.OrderResult)
 	err = tracker.dc.signAndRequest(msgOrder, route, result)
 	if err != nil {
+		details := fmt.Sprintf("cancel: unable to signAndRequest: %v", err)
+		c.notify(newOrderNote("signAndRequest failure", details, db.ErrorLevel, nil))
```

Comment:
Here too, prefer to display this error on the form that the user is interacting with rather than the notification tray.